wordpress-mobile/WordPress-iOS
# Log In: Display new login flow

Fixes #n/a 
WPAuthenticator PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/111

This shows the new Log In flow added in the above WPAuth PR.

Previously, when 'Log In' was selected, the email entry view was immediately displayed. Now when 'Log In' is selected, a button view now appears with available options. These options then display the appropriate view.

---
**Old Log In**
![old_flow](https://user-images.githubusercontent.com/1816888/62981330-2e64da80-bde6-11e9-81d5-feb257efe9c2.png)

**New Log In**
![new_flow](https://user-images.githubusercontent.com/1816888/62981334-315fcb00-bde6-11e9-99a5-46172c096500.png)

-----

**To test:**
- Run the app. Log out if necessary.
- Select 'Log In'.
- A button view should appear with log in options (as shown above).
  - Selecting 'Continue with Google' launches the existing Google auth flow.
  - Selecting log in by site address launches the existing self-hosted flow.
  - Select 'Continue with WordPress.com' shows a slightly modified email entry view.

**WordPress view:**
- The instruction text has been modified.
- 'Log in with Google' has been removed.
- Only the log in by site address option is displayed.

![login_wp](https://user-images.githubusercontent.com/1816888/62981786-6f112380-bde7-11e9-8b2a-a105e137af67.png)

Each login flow still follows their respective paths, so logging in should not be affected.

**Old flow:**
- To verify the old flow still works as expected:
  - In `WordPressAuthenticationManager:initializeWordPressAuthenticator`, change `showNewLoginFlow` to `false`.
  - Rerun the app. The old 'Log In' flow should display (as shown above).

---
Update release notes:
